cloudwatch_alarms/variables.tf

```diff
@@ -0,0 +1,54 @@
+variable "environment" {}
+variable "function" {}
+variable "project" {}
+
+variable "evaluation_period" {
+  default     = 1
+  description = "The number of periods over which data is compared to the specified threshold."
+}
+
+variable "period" {
+  default     = 60
+  description = "The period in seconds over which the specified statistic is applied."
+}
+variable "datapoints_to_alarm" {
+  default     = 1
+  description = "How many datapoints over the threshold to trigger the alarm"
+}
+variable "threshold" {
+  description = "The threshold for the metric above which is will alarm"
```

Comment:
Maybe reword description: "The threshold for the metric above which **an alarm is triggered**"

Reply:
What I've written isn't even proper English. I've changed it to your wording.